modules/circuits/views/nodes/nodes/nodes.php

```diff
@@ -279,20 +279,20 @@ function(c) {
     var ingress_port=$('#ingress_port').val();
     let time_stamp = new Date().toJSON();
     var meican_url="<?php echo $meican_url;?>";
-    //var source_vlan=$('#source_vlan').val();
-    //var destination_vlan=$('#destination_vlan').val();
+    var source_vlan=$('#source_vlan').val();
+    var destination_vlan=$('#destination_vlan').val();
     var latency_required=$('#latency_required').val();
     var bandwidth_required=$('#bandwidth_required').val();
 
-    //if(source_vlan>4096||source_vlan<0){
-    //  alert("source vlan should be between 0-4096");
-    //  return;
-    //}
+    if(source_vlan>4096||source_vlan<0){
```

Comment:
same thing on the vlan range here, vlan 4096 and 0 should not be considered valid values